src/pages/en/guides/deploy/sst.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Deploy your Astro Site to AWS with SST
+description: How to deploy your Astro site to AWS with SST
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+You can deploy an Astro site using [SST](https://sst.dev), an open-source framework for deploying applications to AWS. This is the easiest way to get a fully serverless Astro deployment on AWS with SSR support.
+
+You can also use any additional SST constructs like Cron Jobs, Buckets, Queues, etc while maintaining typesafety.
+
+## Quickstart
+
+1. Create an astro project
+1. Run `npx create-sst@rc`
+1. It should detect that you are using Astro and ask you to confirm.
+1. Once you're ready for deployment you can run `npx sst deploy --stage=production`
+
+If you wish to use any [additional SST constructs](https://docs.sst.dev/) they can be added to `sst.config.ts`. Then take a look at [Resource Binding](https://docs.sst.dev/resource-binding) to understand how to use them in your Astro code.
```

Comment:
`````suggestion
To use any [additional SST constructs](https://docs.sst.dev/), add them to `sst.config.ts`. 

```js
// config example here might be nice?
```

Consult the [SST docs on Resource Binding](https://docs.sst.dev/resource-binding) to understand how to use them in your Astro code.
`````

What would you think about adding an example of what the configuration file should look like with contructs added?